src/main/java/com/manardenza/console/VisualUserMenu.java

```diff
@@ -0,0 +1,67 @@
+package com.manardenza.console;
+
+import com.manardenza.Main;
+import com.manardenza.entity.AbstractObject;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+
+public class VisualUserMenu {
+
+    private org.slf4j.Logger log = LoggerFactory.getLogger(VisualUserMenu.class);
+
+    public static void outputSplitLine() {
+        System.out.println("**************************************************************");
+    }
+
+    public static <E> void printListInConsole(List<String> header, List<E> options) {
+        if (header != null) {
+            header.forEach(System.out::println);
+        }
+        if (options != null) {
+            for (int i = 0; i < options.size(); i++) {
+                System.out.printf("%d %s %n", i + 1, options.get(i).toString());
+            }
+        }
+    }
+
+    public static <E extends AbstractObject> void printMapInConsole(Map<String, List<E>> map) {
+        int key = 1;
+        for (Map.Entry<String, List<E>> entry : map.entrySet()) {
+            System.out.print(key++);
+            System.out.println(" " + entry.getKey());
+            if (entry.getValue().size() == 0) {
+                System.out.println("\tNo available rooms found.");
+            }
+            for (int item = 0; item < entry.getValue().size(); item++) {
+                System.out.print("\t" + (item + 1));
+                System.out.print(" " + entry.getValue().get(item));
+            }
+        }
+    }
+
+
+    public String getValidInputFromUser(String message, InputType type) {
+        System.out.println(message);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)